commands/channel.go

```diff
@@ -312,7 +312,7 @@ func listChannelsCmdF(c client.Client, cmd *cobra.Command, args []string) error
 
 func restoreChannelsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	if len(args) < 1 {
-		return errors.New("Enter at least one channel.")
+		return errors.New("Enter at least one channel")
```

Comment:
Note that the error message has been deliberately modified, otherwise golint throws the following warning.

`error strings should not be capitalized or end with punctuation or a newline go-lint`